Okta/OktaAuth/OktaTokenManager.swift

```diff
@@ -164,25 +179,42 @@ extension OktaTokenManager {
     }
 }
 
-// Handles serialization of OktaAuthConfig.
-// Needed as a temporary solution to omit changing the structre of serialized
-// OktaTokenManager state.
-// TODO: rework this approach in terms of OktaTokenManager refactoring.
-private extension OktaAuthConfig {
+private extension OktaTokenManager {
+    var issuer: String? {
+        return authState.lastAuthorizationResponse.request.configuration.issuer?.path
+    }
+    
+    var clientId: String {
+        return authState.lastAuthorizationResponse.request.clientID
+    }
+    
+    var discoveryDictionary: [String: Any]? {
```

Comment:
Is it possible we can get into a state where this value is different than `OktaAuth.discoveredMetadata`? Seems like we should have `OktaAuth.discoveredMetadata` reference this value to play it safe.

Reply:
This is the same `discoveredMetadata` that is being assigned to `OktaAuth.discoveredMetadata` during auth flow